From armyofthepenguin@gmail.com  Wed Feb 17 15:32:50 2010
From: Rodrigo P. Gomez <armyofthepenguin@gmail.com>
Date: Mon, 8 Feb 2010 16:26:20 -0500
Subject: Staging: new device id for rtl8192e
To: "Greg Kroah-Hartman" <gregkh@suse.de>
Message-ID: <201002081626.20359.armyofthepenguin@gmail.com>

From: Rodrigo P. Gomez <armyofthepenguin@gmail.com>

The rtl8192e driver generates the module r8192e_pci for devices which
have (vendor,pci-id) values (0x10ec, 0x8192), among others.  There is
a new Realtek wireless chipset with pair value (0x10ec,0x8172) which
this driver should work with. This patch adds the new pair to the list
of devices supported.

From: Rodrigo P. Gomez <armyofthepenguin@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rtl8192e/r8192E_core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/rtl8192e/r8192E_core.c
+++ b/drivers/staging/rtl8192e/r8192E_core.c
@@ -107,7 +107,7 @@ static struct pci_device_id rtl8192_pci_
 #else
 	/* Realtek */
 	{ PCI_DEVICE(0x10ec, 0x8192) },
-
+	{ PCI_DEVICE(0x10ec, 0x8172) }, /* Toshiba Satellite 500D */
 	/* Corega */
 	{ PCI_DEVICE(0x07aa, 0x0044) },
 	{ PCI_DEVICE(0x07aa, 0x0047) },
